plugins/woocommerce/tests/php/src/Blocks/BlockTypes/FilledMiniCartContentsBlockTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Tests\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\BlocksSharedState;
+use Automattic\WooCommerce\Tests\Blocks\Helpers\FixtureData;
+use Automattic\WooCommerce\StoreApi\StoreApi;
+
+/**
+ * Tests for the FilledMiniCartContentsBlock block type.
+ */
+class FilledMiniCartContentsBlockTest extends \WP_UnitTestCase {
+
+	/**
+	 * Test products.
+	 *
+	 * @var array
+	 */
+	private $products;
+
+	/**
+	 * Set up test fixtures.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+		StoreApi::container();
+
+		$fixtures       = new FixtureData();
+		$this->products = array(
+			$fixtures->get_simple_product(
+				array(
+					'name'          => 'Test Product',
+					'stock_status'  => 'instock',
+					'regular_price' => 10,
+				)
+			),
+		);
+
+		WC()->cart->empty_cart();
+		add_filter( 'woocommerce_is_rest_api_request', '__return_false', 1 );
+	}
+
+	/**
+	 * Tear down test fixtures.
+	 */
+	public function tearDown(): void {
+		parent::tearDown();
+
+		WC()->cart->empty_cart();
+		remove_filter( 'woocommerce_is_rest_api_request', '__return_false', 1 );
+		remove_all_actions( 'woocommerce_store_api_cart_errors' );
+
+		$this->reset_blocks_shared_state();
+	}
+
+	/**
+	 * Reset the static state in BlocksSharedState between tests.
+	 */
+	private function reset_blocks_shared_state(): void {
+		$reflection = new \ReflectionClass( BlocksSharedState::class );
+		$property   = $reflection->getProperty( 'blocks_shared_cart_state' );
+		$property->setValue( null, null );
+	}
+
+	/**
+	 * @testdox Should render error notices in data-wp-context when cart has errors.
+	 */
+	public function test_renders_error_notices_in_context_when_cart_has_errors(): void {
+		// TODO: Should we hardcode the path? Because server side rendering does not support SVG, we will get a _doing_it_wrong warning,
```

Comment:
@DAreRodz I've added some tests, but I wanted to get your opinion on whether we should hardcode the path of the SVG for now to avoid this issue. It seemed from my perspective that we shouldn't cause warnings in normal usage of WooCommerce.

I also wonder, if you agree it's an issue whether we should check if we're doing directive binding to SVGs anywhere else in WooCommerce.